src/components/sidebar/routesView/routesList.scss

```diff
@@ -2,9 +2,9 @@
 .routesListView {
     display: flex;
     flex-direction: column;
+    overflow-y: auto;
```

Comment:
This breaks sticky Save button

Reply:
Fixed, save button moved to form's parent. TODO's added.